From: Bram Moolenaar <Bram@vim.org>
Date: Sun, 25 Nov 2018 04:25:58 +0100
Subject: patch 8.1.0546: modeline test with keymap fails

Problem:    Modeline test with keymap fails.
Solution:   Check that the keymap feature is available.

(cherry picked from commit 4ace6ab7e72151d78f1563b23b498ed20fa00211)

Signed-off-by: James McCoy <jamessan@debian.org>
---
 src/testdir/test_modeline.vim | 3 +++
 src/version.c                 | 2 ++
 2 files changed, 5 insertions(+)

diff --git a/src/testdir/test_modeline.vim b/src/testdir/test_modeline.vim
index c1eb991..d29bbb5 100644
--- a/src/testdir/test_modeline.vim
+++ b/src/testdir/test_modeline.vim
@@ -44,6 +44,9 @@ func Test_modeline_syntax()
 endfunc
 
 func Test_modeline_keymap()
+  if !has('keymap')
+    return
+  endif
   call writefile(['vim: set keymap=greek :', 'nothing'], 'Xmodeline_keymap')
   let modeline = &modeline
   set modeline
diff --git a/src/version.c b/src/version.c
index bdd0db7..02a53e8 100644
--- a/src/version.c
+++ b/src/version.c
@@ -1195,6 +1195,8 @@ static int included_patches[] =
  */
 static char *(extra_patches[]) =
 {   /* Add your patch description below this line */
+/**/
+    "8.1.0546",
 /**/
     "8.1.0544",
 /**/
